### PR TITLE
enable nncc gbs build

### PR DESCRIPTION
**Pull request by chunseoklee**

### CODE REVIEW

**Comment by seanshpark** [comment]
@chunseoklee , is this just a test or are you planning to post PRs?

**Comment by chunseoklee** [comment]
> @chunseoklee , is this just a test or are you planning to post PRs?

For now, just for test. But I will post PR soon after tiding.

**Comment by seanshpark** [comment]
> But I will post PR soon after tiding.

OK, then please add new `Makefile.*` not modify existing ones.

File: infra/nncc/Makefile.arm32.native [hunk 1]
```diff
@@ -24,10 +24,10 @@ ARM32_BUILD_ITEMS+=;pepper-csv2vec;crew
 ARM32_BUILD_ITEMS+=;oops;pepper-assert
 ARM32_BUILD_ITEMS+=;hermes;hermes-std
 ARM32_BUILD_ITEMS+=;loco;locop;logo-core;logo
-ARM32_BUILD_ITEMS+=;safemain;mio-circle05;
+ARM32_BUILD_ITEMS+=;mio-circle05;
 ARM32_BUILD_ITEMS+=;foder;arser;vconone
 ARM32_BUILD_ITEMS+=;luci;luci-compute
-ARM32_BUILD_ITEMS+=;circle-quantizer
+ARM32_BUILD_ITEMS+=;circle-quantizer-lib
```
Comment (seanshpark) [inline infra/nncc/Makefile.arm32.native]: Q) why are these changed?
